### DISCUSSION TITLE
How to change the height and width of ResizableHandle svg?

### DISCUSSION THREAD

**Discussion by fazlerocks** [shadcn-ui/ui#4269] [Q&A]
Hey folks,

I'm trying to change the SVG size inside ResizableHandle. `w-10 h-10` don't work on the SVG. Any ideas?

**Comment by itaim18** [comment]
I wonder how are you changing the size of the svg, if you're doing it in the `./components/ui/resizable.tsx` at the `ResizableHandle` component: I would rewrite both the svg and the div containing it like this:
```tsx
{withHandle && (
      <div className="z-10 flex h-10 w-10 items-center justify-center rounded-sm border bg-border">
        <GripVertical className="h-10 w-10" />
      </div>
    )}
```
